b2luigi/batch/processes/gbasf2.py

```diff
@@ -531,7 +531,7 @@ def _download_dataset(self):
                 print(stdout)
                 if "No file found" in stdout:
                     raise RuntimeError(f"No output data for gbasf2 project {self.gbasf2_project_name} found.")
-                tmp_output_dir = os.path.join(tmpdir_path, self.gbasf2_project_name)
+                tmp_output_dir = os.path.join(tmpdir_path, self.gbasf2_project_name) + '/sub00'
```

Comment:
Please change this to
```python
os.path.join(tmpdir_path, self.gbasf2_project_name, sub00)'
```
[path.join()](https://docs.python.org/3/library/os.path.html#os.path.join) is a special function for building path names from strings which works across operating systems and handles some special cases intelligently. In this case I think it should make no difference, but since we already have the `join` function in the code, it would be more consistent to put the `sub00` just inside that.